Dockerfile

```
@@ -1,12 +1,27 @@
-ARG SOURCE_BINARY=bin/manager
+FROM golang:1.17 as builder
+
+ARG TAG
+ARG COMMIT
+ARG REPO_INFO
+WORKDIR /workspace
+
+ADD go.mod .
+ADD go.sum .
+RUN go mod download
+
+COPY pkg/ pkg/
+COPY api/ api/
+COPY cmd/ cmd/
+COPY controllers/ controllers/
+COPY main.go main.go
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
```

```suggestion
RUN CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -o manager main.go
```

Remove GOARCH as addon-manager must be build multi-arch.

---

You will also need to update Makefile to build multi-arch via this way of Docker, today it was using goreleaser but the Dockerfile breaks that build.

---

@kevdowney I am confused about the goreleaser command https://github.com/keikoproj/addon-manager/blob/master/Makefile#L103.
Do you mind share your suggestion ?

---

After double check, this change is transparent to goreleaser, the arm64 and amd64 works as previously.

https://github.com/keikoproj/addon-manager/blob/master/.goreleaser.yml#L77
https://github.com/keikoproj/addon-manager/blob/master/.goreleaser.yml#L96